README.md

```diff
@@ -45,6 +45,27 @@ To automagically fix any issues use the `-a` flag:
 
 `bundle exec rubocop -a`
 
+### Integrity checker when on-boarding a service
+
+The `/tools` directory contains a script `./check.rb` which allows us to check whether a service
+has been on-boarded correctly to the self-service app. There are a few steps required:
+
+1. Fully on-board the service (or MSA) to self-service, as per the [team manual instructions](https://verify-team-manual.cloudapps.digital/documentation/support/self-service/onboard-new-service.html)
+2. Make sure the [verify-hub-federation-config]() repository is on master and up-to-date
+3. Login to AWS using the gds-cli
+    - `gds aws verify-prod-a -e` for the production environment
```

Comment:
Technically you have to do the `eval` nonsense, or copy-paste, but maybe everyone knows that now :-)

Reply:
I hope that everyone knows that 🤞